src/Slices/ResourceDetails/UI/Tabs/LogTab/Controls/LogLevelFilter.tsx

```diff
@@ -21,7 +21,7 @@ export const LogLevelFilter: React.FC<Props> = ({ filter, setFilter }) => {
     });
 
   const onSelect = (selection) => {
-    update(selection === null ? undefined : selection);
+    update(selection === null || selection === "" ? undefined : selection);
```

Comment:
<img width="399" alt="image" src="https://github.com/inmanta/web-console/assets/44098050/76810d97-f433-47bd-a4ee-c4b8402c5bc8">

That's the error from BE, so how is setting it to an empty string solve the problem since the error explicitly says it can't be an empty string? I'm confused

Reply:
but what I added is opposite of the setting it to empty string. We use some general purpose TextSelect, and it was simpler to add that condition here that to change given component.

The issue is that when deleting chosen option from select, internal function set selection to empty string and that's why it sent invalid request

Reply:
So, setting it to an empty string removes the param from the API-call? If yes, okay, this can be seen as a solution. 

Otherwise, I'd go for updating the getUrl method to make sure that the query doesn't keep empty filterings, may they be empty strings; null; undefined for the loglevels. 

`web-console\src\Slices\ResourceDetails\Data\ResourceLogs\getUrl.ts`

Reply:
Currently yes, if that general purpose component set the value to empty string the param will be removed from the api-call